lorien/Assets/Icons/arrows_fullscreen.png.import

```diff
@@ -0,0 +1,35 @@
+[remap]
+
+importer="texture"
+type="StreamTexture"
+path="res://.import/arrows_fullscreen.png-9a153672c8107b07d02aa0e7a0edb8da.stex"
+metadata={
+"vram_texture": false
+}
+
+[deps]
+
+source_file="res://Assets/Icons/arrows_fullscreen.png"
```

Comment:
This is a slightly modified version of the `arrows-fullscreen` icon from https://icons.getbootstrap.com/ which is MIT licensed

Reply:
I took all the icons from: https://remixicon.com/. It's better if you use one of them for consitency's sake and because these icons are already credited in the about screen and readme. Just search for "Fullscreen".